### ISSUE TITLE
[Feature] - Be able to disable stoploss

### ISSUE THREAD

**Issue by yepl4** [freqtrade/freqtrade#2226]
* Operating system: Debian Buster
* Python Version: 2.7.16
* CCXT version: 1.18.1068
* Branch: Master
* Last Commit ID: ab3e3797a5703a75437e7c40185bb5dd516073d9
 
It will be great to be able to disable stoploss in the conf or in the strategy file.

I have used "max_open_trades": 1 and "stake_amount": "unlimited" and stoploss = -10

The problem if i use negative value when it buy at different price in the order book this happen:

freqtrade.freqtradebot - WARNING - Unable to create trade: Insufficient funds to create market buy order on market BTC/USDT.Tried to buy amount xxx at rate None.Message: binance Account has insufficient balance for requested action.

I really would like disable stoploss in fact a sell signal will be emit and will be like my stop loss.

**Comment by xmatthias** [comment]
The error you receive has nothing to do with stoploss, but with insufficient balance in stake-currency (USDT, in your case).

Also - removing stoploss means risking 100% of your investment. It is technically disabled by setting it to -1 (or -10) - so i don't see the need for such a feature.

If you're comfortable with that, then that's ok and using `-10` will work (as mentioned above, the error you get has nothing to do with the setting of stoploss  unless you use `stoploss_on_exchange` - which would not work (but since your error is on buy, this does not apply anyway).

Using `-0.10` you will get the same error, since the amount in the account is not sufficient.

Please compare /balance with the "amount" above (and verify that you would have enough to buy the amount (xxx) of BTC - which the error tells me you have not.

**Comment by hroff-1902** [comment]
> * Python Version: 2.7.16

Erhm... what?

**Comment by yepl4** [comment]
sorry with the env activated Python 3.7.3

**Comment by xmatthias** [comment]
I think this was clarified in the slack discussion - stoploss is technically disabled by either `-1` (100% loss) or `-10` (`1000% loss`).

I would not suggest to use this approach, but if you're that confident on your sell-signals, this is the way to go.

The errors you received were not related to the settings of stoploss - and according to our discussion on slack, it's now working, so i'll close this issue.
Feel free to reopen if you still think the error is related to stoploss issues.

**Comment by hroff-1902** [comment]
@xmatthias as I understand in the conversations, the confusing part of the freqtrade business logic in freqtrade/freqtradebot.py is 
```
if self.strategy.stoploss is not None
```
So he/she was amused how to set stoploss to None...
